roles/ceph-client/tasks/pre_requisite.yml

```diff
@@ -30,3 +30,33 @@
   when:
     - cephx
     - copy_admin_key
+
```

Comment:
@andrewschoen I add some code here for coping ceph.conf and ceph.client.admin.keyring to ansible host, so we could remotely manage all deployed cluster in a single ansible controller.

See reference: kubernetes-incubator/kubespray#2347